server/README.md

```diff
@@ -56,55 +56,157 @@ then create it from the correct `.env` file (for example, on the Sensors Admin s
 
 `kubectl create secret generic <your-secret-name> --from-env-file=<path-to-env-file>`
 
-## 4. Deploy changes to the Kubernetes Cluster
+## 4. Deploy and run the smoke tests on Staging
 
 We use [helm](https://helm.sh/docs/) to manage our deployments. Helm allows us to deploy different instances of the application corresponding to dev, staging, and production environments as different 'releases'. Before updating production, we run smoke tests to verify that there's nothing obviously wrong with a release candidate.
 
-### Deploy and run the smoke tests on Staging
-
 1. Run `ssh brave@sensors-admin.brave.coop`
 
 1. cd into the `~/BraveSensor/server` directory and run `git checkout production && git pull origin production` to get the latest version of the helm chart
 
-1. Run the command `helm upgrade staging --set secretName=sensor-staging --set image.tag=<your new container tag> ~/BraveSensor/server/sensor-helm-chart` to deploy the latest version to staging
```

Comment:
@trisapeace is there a reason we don't need to set the image tag when we deploy to staging?

Reply:
That makes sense Theresa, I agree with not copy pasting another image tag, if it takes it from `values.yaml` by default.